clusterloader2/testing/density/config.yaml

```diff
@@ -167,19 +174,20 @@ steps:
     objectBundle:
     - basename: latency-deployment
       objectTemplatePath: deployment.yaml
-- measurements:
+
+- name: Waiting for latency pods to be deleted
+  measurements:
   - Identifier: WaitForRunningLatencyDeployments
     Method: WaitForControlledPodsRunning
     Params:
       action: gather
-- measurements:
```

Comment:
This one is tricky - it works, but technically it's more intuitive to first wait for deletion of pods and then report latencies, rather than doing that in-parallel.

Reply:
That's a really good point. This should be a separate step, thanks for catching!
Done.